tools/scripts/generate_binary_build_matrix.py

```diff
@@ -339,8 +337,34 @@ def generate_wheels_matrix(
             desired_cuda = translate_desired_cuda(gpu_arch_type, gpu_arch_version)
 
             installation_pypi = ""
-            if os == "linux" and gpu_arch_version == "11.7" and channel != "release":
+            # special 11.7 wheels package without dependencies
+            # dependency downloaded via pip install
+            if arch_version == "11.7" and os == "linux" and with_pypi_cudnn == ENABLE:
                 installation_pypi = get_wheel_install_command(channel, gpu_arch_type, desired_cuda, python_version, True)
+                ret.append(
+                    {
+                        "python_version": python_version,
+                        "gpu_arch_type": gpu_arch_type,
+                        "gpu_arch_version": gpu_arch_version,
+                        "desired_cuda": translate_desired_cuda(
+                            gpu_arch_type, gpu_arch_version
+                        ),
+                        "container_image": WHEEL_CONTAINER_IMAGES[arch_version],
+                        "package_type": package_type,
+                        "pytorch_extra_install_requirements":
+                        "nvidia-cuda-runtime-cu11;"
+                        "nvidia-cudnn-cu11==8.5.0.96;"
```

Comment:
How often do these hard coded versions change? We need to remember to update it.

Reply:
Will have to come up with new version for each new CUDA version we introduce. For 1.13.0 this was experimental build, now as we are go forward, we should probably find a way to deal with these builds in a future. This however is outside of the scope of this PR